src/borg/archive.py

```diff
@@ -621,7 +621,10 @@ def restore_attrs(self, path, item, symlink=False, fd=None):
             try:
                 xattr.setxattr(fd or path, k, v, follow_symlinks=False)
             except OSError as e:
-                if e.errno not in (errno.ENOTSUP, errno.EACCES):
+                if e.errno == errno.E2BIG:
+                    logger.warning('Value or key of extended attribute %s in file %s is too big for this filesystem' %
```

Comment:
One last nitpick - then squash + merge.

IIRC most other warnings are formatted like:

```
<filename>: something bad
```

something like -> `%s: [xattr/extended attribute] %s is too big [to extract here / for this filesystem]`